pkmodel/solution.py

```diff
@@ -2,16 +2,80 @@
 # Solution class
 #
 
+import matplotlib.pylab as plt
```

Comment:
I'm not sure if some of the tests are failing due to importing matplotlib.pylab instead of just matplotlib or matplotlib.pyplot (which was used in the tutorial on Day 2)